core/services/job/orm.go

```diff
@@ -698,6 +701,42 @@ func (o *orm) FindJob(ctx context.Context, id int32) (jb Job, err error) {
 	return
 }
 
+// FindJobWithoutSpecErrors returns a job by ID, without loading Spec Errors preloaded
+func (o *orm) FindJobWithoutSpecErrors(id int32) (jb Job, err error) {
+	ctx, cancel := pg.DefaultQueryCtx()
+	defer cancel()
+
+	q := o.q.WithOpts(pg.WithParentCtx(ctx))
```

Comment:
This context generation is unnecessary, `q` already encapsulates the default context

Reply:
Removed the context generation.